src/Command/MailchimpSyncAllMembershipsCommand.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\AdherentMessage\Command\CreateStaticSegmentCommand;
+use AppBundle\Entity\CitizenProjectMembership;
+use AppBundle\Entity\CommitteeMembership;
+use AppBundle\Mailchimp\Synchronisation\Command\AddAdherentToStaticSegmentCommand;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+class MailchimpSyncAllMembershipsCommand extends Command
+{
+    protected static $defaultName = 'mailchimp:sync:all-memberships';
+
+    private const COMMITTEE_TYPE = 'committee';
+    private const CP_TYPE = 'cp';
```

Comment:
what about `CITIZEN_PROJECT_TYPE = 'citizen_project'` ?
just for verbosity

Reply:
done ;)